flags.go

```diff
@@ -22,6 +22,17 @@ var (
 		Usage:    "AVS Operator state retriever address",
 		EnvVar:   envVarPrefix + "OPERATOR_STATE_RETRIEVER_ADDR",
 	}
+	ServiceManagerAddrFlag = cli.StringFlag{
+		Name:     "service-manager-addr",
+		Required: true,
+		Usage:    "AVS Service Manager address",
+		EnvVar:   envVarPrefix + "SERVICE_MANAGER_ADDR",
+	}
+	DontUseAllocationManagerFlag = cli.BoolFlag{
+		Name:   "dont-use-allocation-manager",
+		Usage:  "If set to true, will not use the allocation manager",
```

Comment:
Can you add more context here. Most people (like me) who arent following middleware contract changes have no idea what allocation manager is, so wouldn't know whether to set this to true or not. What does it do, what should it be set to? when? etc

Reply:
https://github.com/Layr-Labs/avs-sync/pull/60/commits/a0269a3a0a34a0b66be5b543258f1afc055a5851

lmk if this makes sense